pom.xml

```diff
@@ -112,6 +114,24 @@
                     <target>1.8</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test-jar</goal>
+                        </goals>
```

Comment:
What does this result in? An `Automatic-Module-Name` in the manifest? If so, why is being applied in the test-jar goal?

Reply:
Yes, just explicitly set a module name in the manifest. Seems `test-jar` goal is not needed, although I did check the result jar had the module name. Now I've removed the `test-jar` goal from it.